.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Spike 
+about: Research, design, investigate, prototype
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Purpose
```

Comment:
Despite what the linter says, these have to be second-level since the rendered template appears in a page that already has an H1.
```suggestion
## Purpose
```